c_src/port_nif_common.c

```diff
@@ -84,6 +110,7 @@ int process_json_common(
     
     int ret = JQ_OK;
     // Set error callback here so that it gets the right env
+    *error_msg_wb = NULL;
```

Comment:
Will this cause leakage? Looks to me that in L61 of **c_src/port_nif_common.c** relies on non-NULL of  `*error_msg_wb` to free the old error string?

Reply:
I see, the outer caller will free it with `erljq_free`